### DISCUSSION TITLE
Is there a way to make `->deletable()` per row?

### DISCUSSION THREAD

**Discussion by kylearch** [awcodes/filament-table-repeater#85] [Q&A] [answered]
I've got a situation where the repeater should always at least one row, but subsequent rows are optional. Even though I can make the rows deletable based on the number of rows present, that applies to all rows. It would be helpful to have a way to apply functions to individual rows instead of the whole repeater.

**Comment by awcodes** [comment, marked as answer]
Just set the min items to 1 and let the validation handle it.

**Reply by kylearch** [reply]
Ha, thanks. I didn't even get to submitting the form while trying to figure this out.
